### ISSUE TITLE
Feature request: ble scan

### ISSUE THREAD

**Issue by DerBroader71** [bill88t/ljinux#73]
The facility to do a WiFi scan is really useful - is there a chance of adding the capability to perform a Bluetooth/BLE scan as well on boards that support it?

**Comment by bill88t** [comment]
Yes, I do plan on supporting all board sensors and  connectivity methods through drivers, it's just that it will take millennia for me to solo all of em.
First I have to finish the njinx webserver and some other stuff and then I will work on it.
I will go ahead and add it in #10.
